start_local_dev.py

```diff
@@ -1,5 +1,4 @@
 from scisim import app
 
 if __name__ == "__main__":
-    app.run()
-
+    app.run(debug=True)
```

Comment:
I believe that debug mode is also enabled if you set the environment variable ENABLE_DEBUG:
https://github.com/sci-sim/sci-sim/blob/dev_cl/scisim/__init__.py#L54